index.html

```diff
@@ -53,5 +53,142 @@ <h1 id="myName">
             </ul>
         </div>
       </section>
+      <section class="works">
+        <div class="recent-works">
+          <h2 id="worksTitle">My Recent Works</h2>  
+        </div>
+        <div id="grid-container">
```

Comment:
- [ ] According to the project requirement you should stick to the design as much as possible like color, spacing, font, etc. Please use the same background to match the Figma design and also please see the horizontal line below the header it is totally different please work on thickness and color.
- [ ] I notice that although you use the same font in style it is different in your design. Please see this issue
### Your design
![his](https://user-images.githubusercontent.com/81727988/236483605-f9381c68-8cba-45fe-aa1b-0fe8b9cbba18.png)

### Figma design
![figma](https://user-images.githubusercontent.com/81727988/236483736-044d8e25-b81a-488b-ae37-1edcc6c70036.png)